Enigmatry.Entry.AspNetCore.Tests.NewtonsoftJson.Tests/Enigmatry.Entry.AspNetCore.Tests.NewtonsoftJson.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <IsPackable>false</IsPackable>
+    <Authors>szi</Authors>
```

Comment:
Don't forget to remove the Authors field, otherwise it will overwrite the values from Directory.Build.Props